bundle/nginx/mime.types

```diff
@@ -0,0 +1,135 @@
+types {
```

Comment:
Did you test websockets if they are working correctly on the bundle?

Reply:
Yes